tests/blockchain/test_blockchain.py

```diff
@@ -1775,9 +1771,9 @@ async def test_conditions(self, empty_blockchain, opcode, with_garbage, bt):
             10, wt.get_new_puzzlehash(), list(blocks[-1].get_included_reward_coins())[0]
         )
         coin1: Coin = tx1.additions()[0]
-        secret_key = wt.get_private_key_for_puzzle_hash(coin1.puzzle_hash)
-        synthetic_secret_key = calculate_synthetic_secret_key(secret_key, DEFAULT_HIDDEN_PUZZLE_HASH)
-        public_key = synthetic_secret_key.get_g1()
+        # secret_key = wt.get_private_key_for_puzzle_hash(coin1.puzzle_hash)
+        # synthetic_secret_key = calculate_synthetic_secret_key(secret_key, DEFAULT_HIDDEN_PUZZLE_HASH)
+        # public_key = synthetic_secret_key.get_g1()
```

Comment:
the goal is to make it easy to enable more logging when debugging in here, instead of realizing you can use this trick and re-implement it. I don't know how to pass flags through `pytest`, but I know how to uncomment code.

Reply:
This isn't logging, right?  If logging it could just use a relevant log level I think.  Would you expect to set a breakpoint here after uncommenting?

If you are interested at any point in adding options to the pytest cli I can share examples etc.  Not ready for that right here since I'm not sure how this code is to be used.

Reply:
sorry, I must have mixed up the threads. I meant to respond here: https://github.com/Chia-Network/chia-blockchain/pull/11453/files#r866760836